### DISCUSSION TITLE
Help in filtering expressions

### DISCUSSION THREAD

**Discussion by jokerigno** [Athou/commafeed#1209] [General]
Hi,

I added some filter based on the example text . Here it is:

`title.contains('peanuts') or title.contains('ep.') or title.contains('le prime pagine di oggi') or content.contains('il podcast quotidiano di matteo bordone') or content.contains('evening post, le storie di oggi')`

But it does not works:

![image](https://github.com/Athou/commafeed/assets/23552762/b3c72950-630a-4d71-8cc1-bbce60a7b43d)

Any hint?

Thanks in advance

**Comment by Athou** [comment]
The filtering expression should reflect what you want to read, i.e. it marks entries as read if it evaluates to `false`.
Since the title of the expanded entry contains 'peanuts', it was not marked as read.

**Comment by jokerigno** [comment]
uhm I don't know if I totally get it: I want to mark as read articles containing the word "peanuts" but if I use the expressione written above the sentence will be "true" so no filtering. Then how should I proceed?

**Reply by Athou** [reply]
If you want to mark as read entries containing "peanuts" or "ep.", the expression should be `not title.contains('peanuts') and not title.contains('ep.')`. That way, you'll only see entries not containing "peanuts" and "ep."

**Reply by jokerigno** [reply]
Thanks now it works :)
